multi_scope_acquisition.py

```diff
@@ -719,7 +728,19 @@ def run_acquisition_bmotion(hdf5_path, toml_path, config_path):
             for motion_index in range(motion_list_size):
                 try:
                     print(f"\nMoving to position {motion_index + 1}/{motion_list_size}...")
-                    selected_mg.move_to_index(motion_index)
+                    try:
+                        selected_mg.move_ml(motion_index)
```

Comment:
The actual motion list move on `MotionGroup` is the `move_ml` method.  The `move_to_index` function is only in the LabView wrapper code, `bapsfdaq_motion_lv`.